frontend/packages/core/src/stories/buttongroup.stories.tsx

```diff
@@ -7,9 +8,6 @@ import { ButtonGroup } from "../button";
 export default {
   title: "Core/Button Group",
   component: ButtonGroup,
-  argTypes: {
-    onClick: { action: "onClick event" },
-  },
```

Comment:
each button in the [array](https://github.com/lyft/clutch/blob/429fa9527cf4d1386cd1117d214502a7ceaa20b8/frontend/workflows/ec2/src/resize-asg.tsx#L70) has an onClick event, and with this code block, events weren't being emitted when a button was clicked.

did some digging and learned we can manually add an action to a component, which I think is a good approach for this case since we're [creating individual button components](https://github.com/lyft/clutch/blob/be8d04d692bc5084a687fa24cfc8dbf2aecc54ed/frontend/packages/core/src/button.tsx#L49) from the array passed in.